interface/weno/adminClass.php

```diff
@@ -0,0 +1,740 @@
+                        `Mail_Address_Line_2` varchar(16) DEFAULT NULL,
+                        `Mail_City` varchar(10) DEFAULT NULL,
+                        `Mail_State` varchar(2) DEFAULT NULL,
+                        `Mail_Zip_Code` int(9) DEFAULT NULL,
+                        `Mail_Phone` int(10) DEFAULT NULL,
+                        `Mail_Fax` int(10) DEFAULT NULL,
+                        `EPCS_Permitted` varchar(5) DEFAULT NULL,
+                        `Accept_NewRx` varchar(4) DEFAULT NULL,
+                        `Accept_Refillresponse` varchar(5) DEFAULT NULL,
+                        `Accept_RxChangeresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Verify` varchar(5) DEFAULT NULL,
+                        `Accept_Cancelrx` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryrequest` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Census` varchar(5) DEFAULT NULL,
+                        `Accept_Resupply` varchar(5) DEFAULT NULL,
+                        PRIMARY KEY (`id`)
+                        ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=11150"
+    		);
+
+          sqlStatement(
+            "CREATE TABLE IF NOT EXISTS `erx_rx_log` (
+                         `id` int(20) NOT NULL AUTO_INCREMENT,
+                         `prescription_id` int(6) NOT NULL,
+                         `date` varchar(25) NOT NULL,
+                         `time` varchar(15) NOT NULL,
+                         `code` int(6) NOT NULL,
+                         `status` text NOT NULL,
+                         `message_id` varchar(100) DEFAULT NULL,
+                         `read` int(1) DEFAULT NULL,
+                         PRIMARY KEY (`id`)
+                          ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=133" 
+          	);
+
+          sqlStatement(
+              "CREATE TABLE IF NOT EXISTS `erx_erx_narcotics` (
+                            `id` int(11) NOT NULL AUTO_INCREMENT,
+                            `drug` varchar(255) NOT NULL,
+                            `dea_number` varchar(5) NOT NULL,
+                            `csa_sch` varchar(2) NOT NULL,
+                            `narc` varchar(2) NOT NULL,
+                            `other_names` varchar(255) NOT NULL,
+                             PRIMARY KEY (`id`)
+                           ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=570"
+          	);
+
+          sqlStatement("ALTER TABLE `users` ADD `weno_prov_id` VARCHAR(15) NULL AFTER `newcrop_user_role`");
+          
+          sqlStatement("ALTER TABLE `prescriptions` ADD `ntx` INT(2) NULL DEFAULT NULL AFTER `prn`, ADD `rtx` INT(2) NULL DEFAULT NULL AFTER `ntx`, ADD `txDate` DATE NOT NULL AFTER `rtx`");
+        
+          sqlStatement("ALTER TABLE `pharmacies` ADD `ncpdp` INT(12) NULL AFTER `email`, ADD `npi` INT(12) NULL AFTER `ncpdp`");
```

Comment:
So help me to understand setup because I will l have to in change the activation instruction.

Moving the alter tables to the upgrade SQL means that either I will need to add a script to the admin class file to do it for them because I am not thinking about tech activating this feature but people who don't know what they are doing at all. 

So, how is this going to work when the user gets to the admin to activate the feature. 

I suppose that the system could prompt the user to click a link to execute the sql_upgrade in a new window. Trying to nail down the logic flow to make this work seamlessly.